### DISCUSSION TITLE
Hello, respected author. I saw the datasets you provided, such as HZ Metro and Beijing Subway, in the appendix of your article. But I downloaded these datasets and found that the number of observations does not match the table in your original text. Could you please explain to me the reason for this? Is there a problem with my calculation method?

### DISCUSSION THREAD

**Discussion by Songbirdw** [SalesforceAIResearch/uni2ts#128] [Q&A]
Hello, respected author. I saw the datasets you provided, such as HZ Metro and Beijing Subway, in the appendix of your article. But I downloaded these datasets and found that the number of observations does not match the table in your original text. Could you please explain to me the reason for this? Is there a problem with my calculation method?

**Comment by chenghaoliu89** [comment]
can you list the number difference here?

**Reply by Songbirdw** [reply]
Thank you for your reply! The differences I found are as follows: for example, for the HZMetro dataset, the total number of observations stated in your original appendix is 146000, but after downloading the data, I found that the number of time series is 80, with 2377 points per time series. Therefore, the total number of observations should be 190160, which is different from the data provided in your original article. In addition, I also found that the number of observations I actually calculated on traffic datasets such as SHMetro and Beijing Subway is not quite the same as the data you provided. But I found it consistent with you on other types of datasets such as weather and finance. May I ask what the reason is? Is it the uniqueness of traffic data or is there a problem with my calculation method? Thank you for your reply and answer!

**Comment by Songbirdw** [comment]
Thank you for your reply! The differences I found are as follows: for example, for the HZMetro dataset, the total number of observations stated in your original appendix is 146000, but after downloading the data, I found that the number of time series is 80, with 2377 points per time series. Therefore, the total number of observations should be 190160, which is different from the data provided in your original article. In addition, I also found that the number of observations I actually calculated on traffic datasets such as SHMetro and Beijing Subway is not quite the same as the data you provided. But I found it consistent with you on other types of datasets such as weather and finance. May I ask what the reason is? Is it the uniqueness of traffic data or is there a problem with my calculation method? Thank you for your reply and answer!
